src/lib/UI/Module/ContentTree/NodeFactory.php

```diff
@@ -264,6 +282,9 @@ private function aggregationResultToArray(TermAggregationResult $aggregationResu
         return $resultsAsArray;
     }
 
+    /**
+     * @return mixed
```

Comment:
Note & out of scope, but nonetheless: I'd rather prefer to stay away from `mixed` data type. The primary issue here is that places that use this value have no way of knowing what they're dealing with - and since we're not using PHPStan rule level 9 (we're using 8) it is allowed to use `mixed` with anything.

Good for now :+1: But consider more strict check in places where this setting is actually used. We might bump ruleset to level 9 to combat this.